music_assistant/providers/snapcast/player.py

```diff
@@ -349,18 +347,12 @@ async def _get_or_create_stream(self, stream_name: str, queue_id: str | None) ->
         if stream := self._get_snapstream(stream_name):
             return stream
 
-        # The control script is used only for music streams in the builtin server
-        # (queue_id is None only for announcement streams).
-        if self.provider._use_builtin_server and queue_id:
-            extra_args = (
-                f"&controlscript={urllib.parse.quote_plus(str(CONTROL_SCRIPT))}"
-                f"&controlscriptparams=--queueid={urllib.parse.quote_plus(queue_id)}%20"
-                f"--api-port={self.mass.webserver.publish_port}%20"
-                f"--streamserver-ip={self.mass.streams.publish_ip}%20"
-                f"--streamserver-port={self.mass.streams.publish_port}"
-            )
-        else:
-            extra_args = ""
+        # Note: The control script feature is currently disabled because snapserver requires
+        # controlscripts to be in /usr/share/snapserver/plug-ins/ but the script is in the
+        # Python package directory. This feature was non-functional from before beta 16 due to a bug
+        # and needs proper implementation before re-enabling.
+        # See: https://github.com/music-assistant/server/issues/4452
+        extra_args = ""
```

Comment:
This completely removes the feature. at least you should have commented it out.

the feature was added because many people were requesting it. I fit now turns out that nobody uses it, that is a shame of development resources spent in adding it.

FYI: This feature enabled the fact that snapweb has full metadata of what is playing and you can control the playback from snapweb and snapcast compatible control clients.